Update UXD accessibility guide (#168)
DockYard/styleguides: ux-dev/accessibility-a11y.md

- [eBay's MIND Patterns](https://ebay.gitbooks.io/mindpatterns/)

## Discovery/Testing/Audits

### Keyboard Testing
Testing your work with a keyboard allows you to ensure that you have distinct focus states and that the tab order is logical.

The following keyboard commands are helpful:
- `tab` to navigate between interactive elements
- `shift + tab` to navigate backwards
- `enter` to activate links and buttons
- `space` to activate checkboxes and buttons

### Screenreader Testing

> All screenreaders will read markup in slightly different ways. Manually testing your work with a screenreader is important because it allows you to confirm that any items that need hidden text have hidden text, that your hidden text makes sense, and that any extraneous information is hidden from the screenreaders.

Reviewer: Slight stylistic suggestion:
```suggestion
All screenreaders will read markup in slightly different ways. Manually testing your work with a screenreader is important because it allows you to confirm that any items that need hidden text have it available, that your hidden text makes sense, and that any extraneous information is hidden from the screenreaders.
```

Possible screenreaders to use to test your work include:

> - VoiceOver (Mac only, activate with `⌘F5`)

Reviewer: ```suggestion
- VoiceOver (Mac only, activate with `⌘ + F5`)
```

For more information, reference [WAI-ARIA's guidelines](https://www.w3.org/TR/wai-aria-practices-1.1/#alertdialog) and [A11ycast's episode on alerts](https://www.youtube.com/watch?v=5lzAj1ahRSI).

## Helpful Tips

### Animation

Animation can be used to help with wayfinding or to surprise and delight users. It's important to use animation in a responsible way as it can trigger [vestibular disorders](https://a11yproject.com/posts/understanding-vestibular-disorders/). For a deep dive, read Val Head's article on [animating responsibly](http://alistapart.com/article/designing-safer-web-animation-for-motion-sensitivity/).

High level tips for animation:
- Be careful of too much motion

> - If there is continous motion, include ways for users to stop and hide the animation ([W3C Criteria 2.2.2](https://www.w3.org/TR/WCAG20-TECHS/F16.html)).

Reviewer: ```suggestion
- If there is continuous motion, include ways for users to stop and hide the animation ([W3C Criteria 2.2.2](https://www.w3.org/TR/WCAG20-TECHS/F16.html)), and strongly consider having the motion paused by default.
```
